### PR TITLE
Documentation: add gdpr non-timeout example

### PR DESCRIPTION
**Pull request by ChrisHuie**
Shows an example of an indefinite timeout on waiting for GDPR consent per issue #5867

### CODE REVIEW

File: integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
+<html>
+  <head>
+    <script async src="../../build/dev/prebid.js"></script>
+    <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
```
Comment (patmmccann) [inline integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html:4]: this is the old gpt

Comment (ChrisHuie) [reply integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html]: updated with new link 🙏

File: integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      </script>
+
+      <script>
+        var googletag = googletag || {};
+        googletag.cmd = googletag.cmd || [];
+        googletag.cmd.push(function() {
+          googletag.pubads().disableInitialLoad();
+        });
+
+        _tcfapi('addEventListener', 2, function(tcData, success) {
```
Comment (ncolletti) [inline integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html:116]: this page is using a different version of a CMP, using window.__cmp.. also the function should be setup as:
`window.__cmp('addEventListener', 'onSubmit', function(tcData, success) {
  if (tcData.event === 'onSubmit') {}
})`

File: integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+          if (tcData.eventStatus === 'useractioncomplete') {
+              var slotCodes = tagSlots
+                  .map( function(slot) { return slot.getSlotElementId() || ''; } )
+                  .filter( function(id) { return id; } );
```
Comment (ncolletti) [inline integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html:120]: Do we need this step? We could likely just do a pbjs.addAdUnits(adUnits) before registering this event listener.

File: integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html (hunk 2 above)
Comment (ChrisHuie) [reply integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html]: Oops. Sorry, I should have relooked at this pr before submitting it. I did it in between a few things. Cleaned it up :)

File: integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html (hunk 3 above)
Comment (ChrisHuie) [reply integrationExamples/gpt/gdpr_indefinite_timeout_exampe.html]: No we dont. Yeah. Got rid of this. Thanks :)

**Pull request merged**